header/src/main/webapp/WEB-INF/ws-servlet.xml

```diff
@@ -27,8 +27,14 @@
         <constructor-arg value="header" />
     </bean>
 
-    <context:property-placeholder
-        location="file:${georchestra.datadir}/default.properties, file:${georchestra.datadir}/header/header.properties"
-        ignore-resource-not-found="true"
-        ignore-unresolvable="true" />
+    <bean id="properties"
+          class="org.springframework.beans.factory.config.PropertyPlaceholderConfigurer">
+        <property name="locations">
+            <list>
+                <value>file:${georchestra.datadir}/default.properties</value>
+                <value>file:${georchestra.datadir}/header/header.properties</value>
+            </list>
+        </property>
+    </bean>
```

Comment:
Not a blocker for me, but what the added value of defining the bean this way and not using the context: namespace ? (I'd expect the end result to be the same, but the modification surprised me.

Reply:
In fact with context:property-placeholder, it does not work , i have tried many changes but useless.

Reply:
weird, it should have worked if it was defined the former way before ?! I am ok with the change anyway, as already said.